src/community/who-uses-dart.md

```diff
@@ -0,0 +1,23 @@
+---
+title: Who uses Dart
+description: Mobile apps and Google web apps are two of the most common uses for the Dart language.
+toc: false
+---
+
+Many Dart developers write mobile apps for Android, iOS, or both,
```

Comment:
I think we have to clarify the connection between Dart and Flutter in this statement

Reply:
Maybe:

The popular Flutter toolkit relies on the Dart language, so developers writing Flutter apps — for Android, iOS, or other targets — use Dart.

Reply:
👍